plugins/main/public/utils/applications.ts

```diff
@@ -734,7 +734,7 @@ const wazuhPluginAbout = {
   category: 'management',
   id: 'wazuh-plugin-about',
```

Comment:
**nitpick**: rename to `about`.

Reply:
done [b1a4838](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6086/commits/b1a483869e88f2ab865dfddf90829d9628153202)